bogota-abril-2019/css-magazine-cover/JuanChaparro/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    <meta charset="UTF-8">
+    <title>Magazine Cover</title>
+    <link rel="stylesheet" type="text/css" href="./styles.css">
+    </head>
+    <body>
+        <div id="container">
+            <div id="main-container">
+                <header>
+                    <h3>April 12, 2010</h3>
+                    <h2><span>Environment Special:</span> The Perils of Plastic</h2>
+                    <h1>TIME</h1>
+                </header>
+                <footer>
+                    <h2>Inside</h2>
```

Comment:
Incorrect use of semantic. If the heach one of the h2 is the same context should only be one

Reply:
Done.